01_fib/01_fib.go

```diff
@@ -0,0 +1,30 @@
+
+package main
+
+import (
+"fmt"
+"io"
+"os"
+)
+
+var out io.Writer = os.Stdout
+
+var (
+	num1 = 0
+	num2 = 1
+	sum = 0
+)
```

Comment:
These should be declared inside `Fib()`.

Reply:
Done